zulipterminal/ui.py

```diff
@@ -208,8 +208,7 @@ def keypress(self, size: urwid_Box, key: str) -> Optional[str]:
             or is_command_key("PRIVATE_MESSAGE", key)
         ):
             self.body.focus_col = 1
-            self.middle_column.keypress(size, key)
-            return key
+            return self.middle_column.keypress(size, key)
```

Comment:
Instead of minor changes, we could point out what the code does exactly in the commit title. :)

Reply:
I've made some changes and can be found at #1072